docs/data/charts/funnel/funnel.md

```diff
@@ -67,6 +67,21 @@ It accepts a number that represents the gap in pixels.
 
 {{"demo": "FunnelGap.js"}}
 
+### Border radius
+
+The border radius of the sections can be customized by the `borderRadius` property.
+It accepts a number that represents the radius in pixels.
+
+- The `bump` curve interpolation will not respect the border radius.
```

Comment:
Why?

Reply:
Couldn't get the math right. Too complex. 

Like, the bump is a quadratic bezier curve like in the left below, with border radius, in the other curves I do something like the right. 

But doing the curve correctly between the two secondary points without weird bumps proved difficult to me. I spent a few hours on it, and then decided it wasn't worth it 😆  

![image](https://github.com/user-attachments/assets/37e78109-5c84-45e6-b570-ce3e489e5324)

Reply:
Sounds good. Should we try to explain to what extent the linear curve respects the border radius? 

From the point of view of someone reading the docs, "the linear curve respects the border radius to some extent" doesn't provide a lot of clarity. Can we be more specific about what are the problems of using a linear curve with the border radius?

Reply:
it is a bit hard to explain to the user. We could try to more closely follow the border radius, but the shape looks odd quite fast, this is 10px radius

<img width="380" alt="Screenshot 2025-05-06 at 13 16 38" src="https://github.com/user-attachments/assets/d6448c70-3e62-44a0-a680-1b4b46fbe342" />

Reply:
I see. Since there's the demo below, I'm ok with keeping it as is, as the user can see for themselves.